web/src/beta/lib/core/engines/Cesium/Sketch/Entity.tsx

```diff
@@ -0,0 +1,43 @@
+import { Entity as CesiumEntity, type EntityCollection } from "@cesium/engine";
+import { omit } from "lodash-es";
+import { forwardRef, memo, useEffect, useState } from "react";
+import { useCesium } from "resium";
+
+import { assignForwardedRef } from "./utils";
+
+export interface EntityProps extends CesiumEntity.ConstructorOptions {
+  entities?: EntityCollection;
+}
+
+export const Entity = memo(
```

Comment:
We already have shared Entity component in [here](https://github.com/reearth/reearth/blob/3bb0a22483a8c75a80791ca5c434ac9ec15fd093/web/src/beta/lib/core/engines/Cesium/Feature/utils.tsx#L74). Can't we use this? This attaches some tags to identify the feature, so selectedFeature should work by this component.

Reply:
Ah I understood. The sketched entity is added as a general feature after an Entity is sketched, right? Then this is no problem, that's make sense :pray:

Reply:
Yes the sketch entities will be used only when drawing. After one feature been created it will be rendered with NLS Feature instead.